addons/product/models/product.py

```diff
@@ -162,6 +162,7 @@ class ProductProduct(models.Model):
 
     pricelist_item_ids = fields.Many2many(
         'product.pricelist.item', 'Pricelist Items', compute='_get_pricelist_items')
+    product_seller_ids = fields.One2many('product.supplierinfo', 'product_id', 'Variant Vendors')
```

Comment:
What happens when product_id is False?  (in case the line is valid for all lines?)